src/NJsonSchema.CodeGeneration.TypeScript/DataConversionGenerator.cs

```diff
@@ -72,22 +74,21 @@ private static object CreateModel(DataConversionParameters parameters)
                 ArrayItemType = parameters.Resolver.TryResolve(parameters.Schema.Item, parameters.TypeNameHint) ?? "any",
                 IsArrayItemNewableObject = parameters.Schema.Item != null && IsNewableObject(parameters.Schema.Item),
                 IsArrayItemDate = IsDate(parameters.Schema.Item?.Format, parameters.Settings.DateTimeType),
+                IsArrayItemDateTime = IsDateTime(parameters.Schema.Item?.Format, parameters.Settings.DateTimeType),
 
                 StringToDateCode = parameters.Settings.DateTimeType == TypeScriptDateTimeType.Date ? "new Date" : "moment",
-                DateToStringCode = "toISOString()", 
+                DateToStringCode = "toISOString().slice(0, 10)",
```

Comment:
Is this safe? Looks a little ugly to me :-)

Reply:
Haha, yeah this works. It's a bit ugly, but it works for `Date` and `moment` types:
```js
moment().toISOString().slice(0, 10)
"2017-10-02"
new Date().toISOString().slice(0, 10)
"2017-10-02"
```
Luckily `toISOString` is well defined :)